projects/js-packages/ai-client/src/data-flow/context.tsx

```diff
@@ -61,7 +61,7 @@ type AiDataContextProviderProps = {
  *
  * @returns {AiDataContextProps} Context.
  */
-export const AiDataContext = createContext( {} as AiDataContextProps );
+export const AiDataContext = createContext( {} ) as React.Context< AiDataContextProps >;
```

Comment:
That is good to know, and I suppose I should have checked before choosing which one to use. I thought the hooks coming from `@wordpress/element` were supposed to be used since they come from an internal library, but if they're untypes and predate typescript, I guess it's better to use the React ones. Thank you for this info